packages/edit-post/src/utils/configure-preferences.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * WordPress dependencies
+ */
+import { dispatch } from '@wordpress/data';
+import { store as preferencesStore } from '@wordpress/preferences';
+import { createDatabasePersistenceLayer } from '@wordpress/wp-preferences';
+
+export default async function configurePreferences( defaults ) {
+	const databasePersistenceLayer = createDatabasePersistenceLayer();
+	await dispatch( preferencesStore ).setPersistenceLayer(
+		databasePersistenceLayer
+	);
```

Comment:
Should this be in edit-post or just an `add_inline_script` for the `preferences` package instead? (kind of like we configure apiFetch which is generic to be wp-specific)